main.py

```diff
@@ -11,4 +11,12 @@ async def hello(ctx):
 
     await ctx.send(f'hello, {author.mention}!')
 
+
+@bot.event
+async def on_message(message):
+    await bot.process_commands(message)
+
+    if message.mentions != []:
```

Comment:
Python checking for emptiness object by this way
```suggestion
    if message.mentions:
```